egs2/jtubespeech/tts1/local/data.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Takaaki Saeki
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=2
+
+ctcscore_pruning_threshold=-0.1
+
+log "$0 $*"
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+if [ -z "${JTUBESPEECH}" ]; then
+   log "Fill the value of 'JTUBESPEECH' of db.sh"
+   exit 1
+fi
+db_root=${JTUBESPEECH}
+
+train_set=tr_no_dev
+train_dev=dev
+recog_set=eval1
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    log "stage 0: Data Download"
+    local/download.sh "${db_root}"
+fi
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "stage 1: local/data_prep.sh"
+    # Initial normalization of the data
+    # Doesn't change sampling frequency and it's done after stages
+    local/data_prep.sh "${db_root}/jtuberaw" "${db_root}/jtubesplit" data/train "${ctcscore_pruning_threshold}"
+fi
```

Comment:
Just confirmation.
Is the audio trimmed well?
If not, why don't you add silence trimming stage?
like:
https://github.com/espnet/espnet/blob/d628250ba5f1885aebe80e246c0791338f5a79e8/egs2/kss/tts1/local/data.sh#L72-L85

Reply:
Thanks! I added the silence trimming stage.

Reply:
OMG This is my mistake.
I will add in another PRs.

Reply:
I added it in #3467.
Sorry for bothering you.